spec/elastic_apm/spies/mongo_spec.rb

```diff
@@ -66,20 +66,54 @@ def start_mongodb
           server_selection_timeout: 5
         )
 
+      # ParallelCollectionScan can only be run on an existing collection.
+      client['testing'].create
       ElasticAPM.with_transaction 'Mongo test' do
-        client['testing'].find.to_a
+        client['testing'].parallel_scan(1).to_a
       end
 
       span, = @intercepted.spans
 
-      expect(span.name).to eq 'find'
+      expect(span.name).to eq 'elastic-apm-test.testing.parallelCollectionScan'
       expect(span.type).to eq 'db.mongodb.query'
       expect(span.duration).to_not be_nil
 
       db = span.context.db
       expect(db.instance).to eq 'elastic-apm-test'
       expect(db.type).to eq 'mongodb'
-      expect(db.statement).to eq 'testing.find'
+      # ParallelCollectionScan doesn't send 'lsid' in the command so we can validate the entire command document.
+      expect(db.statement).to eq "#{BSON::Document.new('parallelCollectionScan' => 'testing', 'numCursors' => 1)}"
```

Comment:
Let's just have the final string in the spec?

Reply:
Sure. I originally put this in case `BSON::Document` serialization to a String did something special.

Reply:
I guess we'd want to know if it did (or started doing)

Reply:
Good point